README.md

```diff
@@ -43,4 +49,10 @@ There are two rules only: respect others and don't forget to pay for yourself in
 Just join us. Usually, we propose one-two topics before the meeting and someone takes role of speaker and prepares some material.
 However, noone likes to prepare and often we just extemporize. As an option, two or more topics may be discussed at the same time.
 
+## Why thursday?
+We spend friday nights with our families and our closest friends. Whereas thursday is almost a week end, and the evening is usually free.
+
```

Comment:
## Why Thursday?
We spend Friday nights with family and closest friends, whereas Thursday is almost weekend and the evening is usually free.